pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -129,10 +133,23 @@ func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus) erro
 	}
 
 	cephCluster.Status.CephStatus = toCustomResourceStatus(cephCluster.Status, status)
+
 	if err := opcontroller.UpdateStatus(c.client, cephCluster); err != nil {
 		return errors.Wrapf(err, "failed to update cluster %q status", clusterName.Namespace)
 	}
 
+	var condition cephv1.ConditionType
+	var reason, message string
+	condition = cephv1.ConditionReady
```

Comment:
nit: no need to define these on a separate line. The same applies to `reason` and `message`.
```suggestion
	condition := cephv1.ConditionReady 
```